src/sdk.js

```diff
@@ -1,57 +1,111 @@
 window.Smartcar = (function(window) {
   'use strict';
+  // accesss denied error class
+  class AccessDenied extends Error { // eslint-disable-line no-restricted-syntax
+    constructor(message) {
+      super(message);
+      this.name = 'AccessDenied';
+    }
+  }
 
   /**
    * Initializes Smartcar class
    *
    * @param {Object} options - the SDK configuration object
    * @param {String} options.clientId - the applications' client id
-   * @param {String} options.redirectUri - the registered redirect uri of the application
+   * @param {String} options.redirectUri - the registered redirect uri of the
+   * application
    * @param {String[]} [options.scope] - requested permission scopes
-   * @param {Function} [options.onComplete] - called upon completion of the Auth flow
-   * @param {Boolean} [options.development=false] - launch Smartcar auth in development mode
-   * to enable the mock vehicle brand
+   * @param {Function} [options.onComplete] - called on completion of auth flow
```

Comment:
nit: `of auth flow` --> `of the Smartcar auth flow`

Reply:
This seems unnecessary given that it's in Smartcar source code? What's the reasoning for adding it, avoiding confusion with another auth flow?